.github/workflows/main.yml

```diff
@@ -0,0 +1,71 @@
+# The following configuration is used to setup GitHub Actions
+# for building (and running unit tests) on pushes and PRs.
+# At the moment, we build on two runner types: Linux and MacOS,
+# specific version information can be found further below.
+name: Build On Changes
+
+# run action on all pushes and pull_request for all branches
+on: [push, pull_request]
+
+# high-level constants
+env:
+  BASE_URL: https://gitlab.science.ru.nl/sac-group/sac-packages/-/raw/master/packages/weekly
+  VERS: 1.3.3-383-1
+  SPKG: sac2c-1.3.3-MijasCosta-383-g6ea12-omnibus
+
+# action listing, all jobs run in parallel
+jobs:
+  build_linux:
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Get HEAD and submodules
+        uses: actions/checkout@v2
+        with:
+          # we need to get tag information
+          fetch-depth: 0
+          # we need to get all submodules
+          submodules: 'recursive'
+      - name: Install sac2c-basic
+        run: |
+          wget ${BASE_URL}/Ubl18/${VERS}/basic/${SPKG}.deb
+          sudo apt install ./${SPKG}.deb
+          sac2c -V
+      - name: Create build dir
+        run: |
+          cmake -E make_directory ${{runner.workspace}}/build
+          mkdir $HOME/.sac2crc
```

Comment:
Isn't this done automatically by the buildsystem?

Reply:
No, we don't do this intentionally: https://github.com/SacBase/cmake-common/blob/b71932461a002f20dc3e6dfcff91aee5950cf18c/generate-sac2crc-file.cmake#L56.

Reply:
Really weird, if I think about it.  I could understand that if you find `.sac2crc` file, and not the directory, you refuse to move it.  But if there is no directory and no file, why wouldn't one create it?  But ok, this is not really relevant to this PR.

Reply:
Shall we open an issue in cmake-common?   Or there is any reason to keep this behaviour?

Reply:
No, I think we just kinda copied it over the sac2c build system... maybe we should change this in both cases, because well its a bit silly.

Reply:
Sounds good to me.  Just open an issue/pr in stdlib that acknowledges that we want to create the directory in case it is not there, and let's merge this pr.

Reply:
Ok, created PR #38.